tinydrive/tinydrive-api/standalone.md

```diff
@@ -51,10 +74,38 @@ The URL for the selected file would be in the following format: `https://drive.t
 
 The modification date for the file in ISO 8601 format for example `2019-02-24T15:00:00Z`
 
+## `tinydrive.controls.pick`
+
+The `tinydrive.controls.pick` method enables you to pick files from {{site.cloudfilemanager}} and get the meta data of those files returned in a callback. For a complete list of available settings to pass into this method check the [Picker settings](#pickersettings) section in this page. A control object is returned which control the {{site.cloudfilemanager}} lifecycle, which it no longer does by itself.
```

Comment:
* `meta data` should be `metadata`
* I'd drop "section in this page" and just say `For ... see [Picker settings](#pickersettings)`, as I think that's more in line with the rest of the docs
* The lifecycle sentence needs some reworking. Firstly, it kinda sounds like we're breaking semver with this change by expecting people to add lifecycle management to their code in order to get the same behaviour they would have had before the upgrade. Secondly, maybe phrase it as something more like `The function returns a control object which is used to control the {{site.cloudfilemanager}} lifecycle. For information on controlling the lifecycle see ${link to paragraph where we discuss what happened with the lifecycle change, what people need to do to fix the breakages when they update their apps, what's going on with the setTimeouts, and what methods are available on the control object}`

Reply:
- We use `meta data`for `tinydrive.pick`. Are you sure it should be `metadata`?
- Sure, though again, that's how it is in `tinydrive.pick`
- This should not be breaking semver. This adds a new option for them to use if they want to control the life cycle. If they continue to use the old functions everything should continue to work as it already is, no API changes to that.

Reply:
I'm 99% sure it should be `metadata`. MDN uses a single word for comparison: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta